src/Compilers/VisualBasic/Test/Syntax/Parser/ParseStatements.vb

```diff
@@ -8433,13 +8433,61 @@ Module M
     Sub Main() _
  _
     _
-_ 
+_
     End Sub
 End Module
 ]]>,
     Diagnostic(ERRID.ERR_LineContWithCommentOrNoPrecSpace, "_").WithLocation(6, 1)
 )
     End Sub
+    <Fact()>
+    <WorkItem(65, "https://github.com/dotnet/vblang/issues/65")>
+    Public Sub ParseLineContWithCommentV16()
+        ' PROTOTYPE LanguageVersion.Latest should be LanguageVersion.VisualBasic16
+        ParseAndVerify(
+        <![CDATA[
+    Module M
+           Sub Main()
+                Dim I As Integer _ ' Comment
+                    = 1
+            End Sub
+     End Module
+    ]]>, New VisualBasicParseOptions(LanguageVersion.Latest)
+        )
+    End Sub
+
+    ' PROTOTYPE LanguageVersion.VisualBasic15_3 should be LanguageVersion.VisualBasic15_5
+    <Fact()>
+    <WorkItem(65, "https://github.com/dotnet/vblang/issues/65")>
+    Public Sub ParseLineContWithCommentV15_5()
+        ParseAndVerify(
+        <![CDATA[
+    Module M
+           Sub Main()
+                Dim I As Integer _ ' Comment
+                    = 1
+            End Sub
+     End Module
+    ]]>, New VisualBasicParseOptions(LanguageVersion.VisualBasic15_3),
+            Diagnostic(ERRID.ERR_LineContWithCommentOrNoPrecSpace, "_").WithLocation(4, 34),
+            Diagnostic(ERRID.ERR_Syntax, "=").WithLocation(5, 21)
+        )
+    End Sub
```

Comment:
i feel like a lot more cases should be tested.  basically, there are a lot of checks and cases in your scanner code.  so we should try to have a bunch of tests to validate all the different types of cases your code checks for.

Reply:
I agree I will work on additional checks like multiple spaces, but I think the code should also separate ERRID.ERR_LineContWithCommentOrNoPrecSpace into 2 or more errors, which is going to be a bigger change. One that is just for NoPrecSpace which is not changing and one that tells people who are using new compiler but old version flag to upgrade and maybe by V16 the IDE will have support to do the upgrade for VB like it does for C#.

Reply:
When I do git status I get
```
On branch vblang-65
Your branch is up to date with 'origin/vblang-65'.

No commands done.
Next command to do (1 remaining command):
   noop
  (use "git rebase --edit-todo" to view and edit)
You are currently editing a commit while rebasing branch 'vblang-65' on '80149cc703'.
  (use "git commit --amend" to amend the current commit)
  (use "git rebase --continue" once you are satisfied with your changes)

nothing to commit, working tree clean
```
Should I do git rebase --continue?

Reply:
Are you doing a rebase?  I'm really not sure what repo steps you're in the middle of.  So it's hard for me to comment on what the right thign is to do here.  We can chat more on gitter if that would be helpful.

Reply:
I review my latest code and compared to original and the only logic I changed is below. The first "If" checked for the one very specific condition where my new code is invoked, an _ followed by exactly one space, followed by a single quote (which might be Unicode). If exactly that pattern is not found I revert to the original code (though I did make some variable have wider scope) with all the original test. If that pattern is found I then check for availability of the feature,  @jcouv I do this second because I hope in the future to have a better error message telling the user to upgrade compiler version. If I get past the second "If" I am in my new code and have tests for that and am happy to add more if necessary. All the functions I call in my code are tested elsewhere and I did not change, even the sequence the functions are called are exactly the same as used elsewhere. Only 4 compiler tests originally tested for this condition and I have updated those to test old and new compiler version. There is one IDE test RemoveUnnecessaryLineContinuationTests.ColonToken_LineContinuation_Comment_BeforeColonToken failing that I don't yet understand well enough to fix.
```
           If CanGet(Here + 1) AndAlso IsWhitespace(Peek(Here)) AndAlso IsSingleQuote(Peek(Here + 1)) Then
                If CheckFeatureAvailability(Feature.CommentsAfterLineContinuation) Then
                    tList.Add(MakeLineContinuationTrivia(GetText(1)))
                    tList.Add(MakeWhiteSpaceTrivia(GetText(Here)))
                    ScanCommentIfAny(tList)
                    ch = Peek()
                    atNewLine = IsNewLine(ch)
                    FoundCommentAfterUnderscore = True
                End If
            End If
            If Not FoundCommentAfterUnderscore Then
```